app/templates/components/public/speaker-list.hbs

```diff
@@ -1,6 +1,6 @@
 <h3>{{t 'Speakers'}}</h3>
-<div class="speaker-list ui grid">
+<div class="speaker-list ui stackable grid">
   {{#each speakers as |speaker|}}
-    {{public/speaker-item speaker=speaker}}
+    {{public/speaker-item speaker=speaker isFeatured=false}}
```

Comment:
@geekyd in Ember.js an undefined parameter is by default `null`. So that will be resolved as a false by any if statement. `isFeatured=false` is not needed.